### PR TITLE
All Languages - update Template S3 Terraform Backend - pages/public_c…

### LINKED ISSUE #5430: documentation for Terraform S3 Backend outdated
Hello,
I've just updated my Terraform version to `v1.6.4` and received an error when using OVH S3 Object Storage as a backend. The backend is configured as explained in [the documentation](https://help.ovhcloud.com/csm/en-public-cloud-compute-terraform-high-perf-object-storage-backend-state?id=kb_article_view&sysparm_article=KB0051345#initializing-terraform-configuration). Also switching back to version `v1.5.7`, it works again, so it's not a problem with my configuration.
The error looks like this:
Screenshot_20231125_121428

Checking out [the Terraform S3 Backend documentation](https://developer.hashicorp.com/terraform/language/settings/backends/s3#s3-1), I was able to fix the configuration:
Screenshot_20231125_121714
With this new configuration it works without any warnings/errors.

The documentation should be updated.

Best regards,
Maximilian Müller

### PR DESCRIPTION
**Pull request by Mx7ca**
…loud/compute/use_object_storage_terraform_backend_state

This PR fixes https://github.com/ovh/docs/issues/5430 in the documentation of all existing languages.

### CODE REVIEW

**Review by scraly — APPROVED** [review]
Top, thank you!

**Comment by scraly** [comment]
Thanks for the PR, I will update it to add the information for old terraform version

**Review by scraly — CHANGES_REQUESTED** [review]
add compatibility with old version

**Comment by scraly** [comment]
The modification will only work for new version, not old version. 

Please add this change:


Before terraform version 1.6.0:

```yaml
terraform {
    backend "s3" {
      bucket = "terraform-state-hp"
      key    = "terraform.tfstate"
      region = "gra"
 #or sbg or any activated high performance storage region
      endpoint = "s3.gra.perf.cloud.ovh.net"
      skip_credentials_validation = true
      skip_region_validation = true
    }
}
```

After terraform version 1.6.0:

```yaml
terraform {
    backend "s3" {
      bucket = "terraform-state-hp"
      key    = "terraform.tfstate"
      region = "gra"
      # sbg or any activated high performance storage region
      endpoints = {
        s3 = "https://s3.gra.perf.cloud.ovh.net/"
      }
      skip_credentials_validation = true
      skip_region_validation      = true
      skip_requesting_account_id  = true
    }
}
```

**Comment by scraly** [comment]
FYI the information about the terraform version is in the hashicorp doc:
https://developer.hashicorp.com/terraform/language/upgrade-guides#s3-backend

**Comment by Mx7ca** [comment]
@scraly thanks for the link to [OVH's documentation](https://developer.hashicorp.com/terraform/language/upgrade-guides#s3-backend). I've updated the version restriction to all files and also updated the "updated" date again.
Is it now okay?

**Comment by scraly** [comment]
Thanks, it's good for me :-) 

The only thing to change is the date of the modification :)

**Review by scraly — APPROVED** [review]

File: pages/public_cloud/compute/use_object_storage_terraform_backend_state/guide.de-de.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Using OVHcloud Object Storage as Terraform Backend to store your Terraform state (EN)'
 excerpt: 'Find out how to use an OVHcloud Object Storage as a Terraform Backend to store your Terraform state'
-updated: 2023-07-31
+updated: 2023-11-30
```
Comment (OVH-Team-Guides) [inline pages/public_cloud/compute/use_object_storage_terraform_backend_state/guide.de-de.md:4]:
```suggestion
updated: 2023-12-04
```
@Mx7ca , could you please update this date on all the versions ? Thank you

Comment (OVH-Team-Guides) [reply pages/public_cloud/compute/use_object_storage_terraform_backend_state/guide.de-de.md:4]: We will do it on our side

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @Mx7ca

**Pull request merged**